src/sentry/static/sentry/app/views/settings/project/serviceHookSettingsForm.jsx

```diff
@@ -13,20 +12,18 @@ import {Panel, PanelBody, PanelHeader} from 'app/components/panels';
 
 const EVENT_CHOICES = ['event.alert', 'event.created'].map(e => [e, e]);
 
-export default createReactClass({
-  displayName: 'ServiceHookSettingsForm',
-
-  propTypes: {
+export default class ServiceHookSettingsForm extends React.Component {
+  static propTypes = {
     orgId: PropTypes.string.isRequired,
     projectId: PropTypes.string.isRequired,
     hookId: PropTypes.string,
     initialData: PropTypes.object.isRequired,
-  },
+  };
 
-  onSubmitSuccess() {
+  onSubmitSuccess = () => {
```

Comment:
github's being weird, on L42 there's an `onCancel` method should that be removed?